lightning/src/offers/offer.rs

```diff
@@ -169,7 +170,7 @@ impl<'a, T: secp256k1::Signing> OfferBuilder<'a, DerivedMetadata, T> {
 		secp_ctx: &'a Secp256k1<T>
 	) -> Self where ES::Target: EntropySource {
 		let nonce = Nonce::from_entropy_source(entropy_source);
-		let derivation_material = MetadataMaterial::new(nonce, expanded_key, IV_BYTES);
+		let derivation_material = MetadataMaterial::new(nonce, expanded_key, IV_BYTES, None);
```

Comment:
IIUC, it can go into the `BlindedPath`s inside the `Bolt12Invoice`. That's how we do stateless inbound payments for offers, at least. We could include additional data there.

Reply:
Def, added tracking for this to #1970.